src/Mmanos/Billing/Gateways/Stripe/Customer.php

```diff
@@ -75,7 +75,7 @@ public function info()
 			$this->stripe_customer = Stripe_Customer::retrieve($this->id);
 		}
 		
-		if (!$this->stripe_customer || $this->stripe_customer->deleted) {
+		if (!$this->stripe_customer || property_exists($this->stripe_customer, 'deleted')) {
```

Comment:
Should you check for both the existence of this property and that it's value is true?